### ISSUE TITLE
What happens if a handle gets revoked?

### ISSUE THREAD

**Issue by agentjabsco** [bluesky-social/social-app#1035]
I'm going to test how Bluesky currently handles this in a moment (I have a hypothesis, but I'm not gonna speak to it until I've determined the actual observed behavior), but this is a subject I've thought about: how should AT Protocol sites handle checking that a user's domain handle is still valid, and what should they do if it isn't?

It shouldn't just be a one-time thing: say, for example, a hellthread.vet user did something to dishonor the Hellthread. In a case like that, it should be possible to strip them of their "title" (note that, although I am referencing their culture for this example, I do not claim to speak for the Hellthreaders, who are a proud and powerful people)

There are other, more serious examples I could draw from here, along the same lines as why an employee at a company shouldn't still have access to their company email address after leaving the company.

Anyway, I will say that this is one reason I don't think Bluesky should release a user's \*.bsky.social handle in the event that they change their presented handle to a different domain. just my 2¢

**Comment by agentjabsco** [comment]
Yeah, as I suspected, users are still identified with their original handle even after the domain record that verified it is changed to point somewhere else. Feels like the kind of thing that could potentially end up leading to problems down the line

**Comment by pfrazee** [comment]
I'm going to bounce this issue to the atproto repo, but I can tell you there's some work being done on dealing with invalid handles (it's specced and in a backend PR and I think frontend just needs to implement). The behaviors around invalidation might need some further discussion (Im not sure where they're at on that)
